Library/Homebrew/rubocops/lines.rb

```diff
@@ -473,10 +473,9 @@ def audit_formula(_node, _class_node, _parent_class_node, body_node)
           end
 
           find_instance_method_call(body_node, "ARGV", :include?) do |method|
-            param = parameters(method).first
-            next unless match = regex_match_group(param, /^--(HEAD|devel)/)
+            next unless parameters_passed?(method, "--HEAD")
 
-            problem "Use \"if build.#{match[1].downcase}?\" instead"
+            problem "Use \"if build.head?\" instead"
```

Comment:
This broke a style test. Instead, just remove the `--devel` sections. You can reproduce with `brew tests --only=rubocops/lines`.

Reply:
I actually think that the new check is better. Before, the problematic node was just the `"HEAD"` string. Now, it's the whole `ARGV.include? "HEAD"` method. In practice, a user will replace the entire method with `build.with? "HEAD"` rather than just the string `"HEAD"`, so I think this way makes more sense.

The test itself still needs to be updated, though.